src/sql/src/session/vars.rs

```diff
@@ -2441,7 +2441,7 @@ impl Var for BuildInfo {
     }
 
     fn value(&self) -> String {
-        self.human_version()
+        self.human_version(None)
```

Comment:
This is the only one that seems a bit off to me—I think this is `SHOW mz_version`, right? And that seems like it ought to print the self-hosted version.

Reply:
Apparently not:
```
materialize=> select mz_version();
                    mz_version
--------------------------------------------------
 v0.124.0-dev.0 (000000000, self-hosted: v25.1.0)
(1 row)
```

Reply:
```
materialize=> show mz_version;
                   mz_version
-------------------------------------------------
 v0.124.0-dev.0 (000000000, helm chart: v25.1.0)
(1 row)

materialize=> select mz_version();
                   mz_version
-------------------------------------------------
 v0.124.0-dev.0 (000000000, helm chart: v25.1.0)
(1 row)
```
Fixed